src/SprykerEco/Zed/UnzerGui/Communication/Plugin/MerchantGui/MerchantUnzerFormExpanderPlugin.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEco\Zed\UnzerGui\Communication\Plugin\MerchantGui;
+
+use Spryker\Zed\Kernel\Communication\AbstractPlugin;
+use Spryker\Zed\MerchantGuiExtension\Dependency\Plugin\MerchantFormExpanderPluginInterface;
+use Symfony\Component\Form\FormBuilderInterface;
+
+/**
+ * @method \SprykerEco\Zed\UnzerGui\Communication\UnzerGuiCommunicationFactory getFactory()
+ */
+class MerchantUnzerFormExpanderPlugin extends AbstractPlugin implements MerchantFormExpanderPluginInterface
+{
+    /**
+     * {@inheritDoc}
```

Comment:
Please, add description for this plugin.

Reply:
Description already given in MerchantFormTabExpanderPluginInterface